style/portfolio.css

```diff
@@ -0,0 +1,37 @@
+/* .viewing-party {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    text-align: center;
+    border-style: solid;
+
+} */
+
+
+.viewing-party,
+.task-list,
+.swap-meet {
```

Comment:
since each `div` uses the same styling, we should name the class all the same thing, like `project-container`.

No reason to create 3 different class names that use the exact same styling. Think about if you have 50 projects; you'd have to name each one differently and add them all to css.